config_defaults_inc.php

```diff
@@ -3618,6 +3618,15 @@
  */
 $g_my_view_boxes_fixed_position = ON;
 
+/**
+ * Toggle whether 'Timeline' boxes show summary of issue after issue ID
+ */
+$g_timeline_show_issue_summary = OFF;
+
+/**
+ * Control the maximum length of each summary shown in timeline when $g_timeline_show_issue_summary is ON.
+ */
+$g_timeline_issue_summary_length = 20;
```

Comment:
There is no need to change anything in SOAP API.
You have just to add the option to array $g_public_config_names in config_defaults_inc.php.
Keep in mind that the current entries are alphabetically sorted.